signac/contrib/project.py

```diff
@@ -41,7 +41,7 @@
 from .indexing import _SignacProjectCrawler
 from .job import Job
 from .schema import ProjectSchema
-from .utility import _mkdir_p, _nested_dicts_to_dotted_keys, split_and_print_progress
+from .utility import _mkdir_p, _nested_dicts_to_dotted_keys, _split_and_print_progress
```

Comment:
If I've had to guess the function was introduced prior to having the tqdm dependency or because there was some kind of threading issue or so with tqdm. But I agree, we should certainly try to replace it with tqdm if possible.